test/browser/server/src/App.vue

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
+
+import {
+  Database,
+  Validator,
+  Registry,
+  helpers
+} from "../../../../dist/esm/index.js";
+import { getDefaultProvider, Wallet } from "ethers";
+
+const wallet = new Wallet(
+  "c526ee95bf44d8fc405a158bb884d9d1238d99f0612e9f33d006bb0789009aaa",
+  getDefaultProvider("http://127.0.0.1:8545")
+);
+const db = new Database({
+  signer: wallet,
+  baseUrl: helpers.getBaseUrl("local-tableland"),
+  autoWait: true,
+});
+
+const validator = new Validator(db.config);
+const registry = new Registry(db.config);
+
+export default {
+  data: function () {
+    return {
+      tablename: "",
+      tableId: "",
+      message: "",
+      statement: "",
+      successMsg: "",
+      responseJson: ""
+    };
+  },
+
+  methods: {
+    runSql: async function (statementArg, successArg) {
+      try {
+        const statementStr = (statementArg || this.$data.statement).trim();
+        const successMsg = (successArg || this.$data.successMsg).trim();
+
+        if (!statementStr) throw new Error("no statement available");
+        if (!successMsg) throw new Error("no success message available");
+
+        const res = await db
+          .prepare(statementStr)
+          .all();
+
+        this.$data.message = successMsg;
+        this.$data.responseJson = JSON.stringify(res);
```

Comment:
The tests interact with the app via a headless browser and then expect the html content to be changed.  By using a simple text message and a stringified object we can test everything we need to.